packages/ds-medicare-gov/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { HelpDrawer, CloseIconThin } from '@cmsgov/design-system';
+
+HelpDrawer.defaultProps = {
```

Comment:
This is my first time making an "mgov component" so I hope this is the right way to do it. This HelpDrawer should include the correct close button/icon when used.